src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -3361,11 +3360,11 @@ void CEndlessUsbToolDlg::StartInstallationProcess()
 			}
 		} else {
 			// installer image file + signature
-			urlInstaller = CString(RELEASE_JSON_URLPATH) + m_installerImage.urlFile;
+			urlInstaller = CString(IMAGE_SERVER_BASE) + m_installerImage.urlFile;
```

Comment:
Since `urlInstaller` (and, below, `urlInstallerAsc`) is used in an `urls` array below, shouldn’t it be *not* prefixing with the base URI?

Reply:
good catch!